src/ngraph/frontend/onnx_import/op/unsqueeze.cpp

```diff
@@ -35,31 +28,11 @@ namespace ngraph
             {
                 NodeVector unsqueeze(const Node& node)
                 {
-                    NodeVector inputs{node.get_ng_inputs()};
-                    auto data = inputs.at(0);
-                    auto data_shape = data->get_shape();
-                    auto axes = node.get_attribute_value<std::vector<std::int64_t>>("axes");
-
-                    ASSERT_VALID_ARGUMENT(node, !axes.empty()) << "'axes' attribute is mandatory.";
-                    ASSERT_VALID_ARGUMENT(
-                        node,
-                        axes.size() ==
-                            std::set<std::int64_t>(std::begin(axes), std::end(axes)).size())
-                        << "'axes' has a duplicate axis.";
-
-                    std::sort(std::begin(axes), std::end(axes), std::less<int64_t>());
-
-                    AxisVector input_order{ngraph::get_default_order(data_shape.size())};
-
-                    for (auto axis : axes)
-                    {
-                        ASSERT_VALID_ARGUMENT(node, axis >= 0 && axis <= data_shape.size())
-                            << "provided 'axes' attribute is not valid.";
-
-                        data_shape.insert(std::next(std::begin(data_shape), axis), 1);
-                    }
-
-                    return {std::make_shared<ngraph::op::Reshape>(data, input_order, data_shape)};
+                    auto data = node.get_ng_inputs().at(0);
+                    auto axes = node.get_attribute_value<std::vector<std::int64_t>>("axes", {});
+                    auto axes_node = std::make_shared<ngraph::op::Constant>(
+                        element::u64, Shape{axes.size()}, axes);
```

Comment:
Is it intended to use `unsigned int` as element type for `axes_node` while `axes` is `signed int`?